modules/oe_content_tender/config/install/field.field.node.oe_tender.oe_departments.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_departments
+    - node.type.oe_tender
+  module:
+    - rdf_skos
+id: node.oe_tender.oe_departments
+field_name: oe_departments
+entity_type: node
+bundle: oe_tender
+label: 'Responsible department'
```

Comment:
I'm not sure if the label of this should not be "Departments", and then oe_author is missing and has this label, but I posted this questions in the ticket so ignore this comment for now, just here for history

Reply:
This requires clarification as "Responsible department" name is colliding with the oe_author field.

Reply:
this was clarified, this label seems to be ok here (is probably wrong in events) and oe_author will not be added to tender content type